ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-release-4.13.yaml

```diff
@@ -90,15 +90,15 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-azure
-- as: e2e-upgrade
+- as: e2e-upgrade-ovn
```

Comment:
for consistency, you should name this e2e-aws-ovn-upgrade

Reply:
oh, applies to the other files as well

Reply:
Only in master, 4.12, and 4.13, correct?  Or all versions?

Reply:
yeah, all versions.

the naming convention that sort of just came about I guess, is "e2e-<platform>-<cni>-<other related tags>" this
one is using the openshift-upgrade-aws , so you'd want "e2e-aws-ovn-upgrade"

Reply:
Done.  For some reason it was as it should be in some of the older files already.